records_mover/records/mover.py

```diff
@@ -106,7 +106,7 @@ def move(records_source: Union[RecordsSource, ContextManager[RecordsSource]],
         target_records_format: BaseRecordsFormat = getattr(records_target, "records_format", None)
         logger.info(f"Mover: copying from {records_source} to {records_target} "
                     f"by first writing {records_source} to {target_records_format} "
-                    "records format...")
+                    "records format (if easy to rewrite)...")
```

Comment:
A little disambiguation on what's going on that helped as I worked on debugging integration tests..